tests/rptest/tests/self_test_test.py

```diff
@@ -99,10 +122,14 @@ def test_self_test_node_crash(self):
             assert node['status'] == 'idle'
             assert node.get('results') is not None
             for report in node['results']:
-                assert 'error' not in report
-                assert 'warning' not in report
-                assert 'duration' in report
-                assert report['duration'] >= 1000, report['duration']
+                # Errors related to crashed node are allowed, for example a network test on a good node that had attempted to connect to the crashed node
+                if 'error' in report:
+                    report[
```

Comment:
missed assert?

Reply:
This is a test that crashes a node so a failure can be expected